requirements.txt

```diff
@@ -3,13 +3,13 @@ docopt == 0.6.2
 mediawiki-utilities==0.4.14
 mwparserfromhell == 0.3.3
 nltk == 3.0.1
-nose == 1.3.4
-numpy == 1.9.2
+nose >= 1.3.4
+numpy >= 1.9.2
 pytz == 2015.4
 requests >= 2.0.0, < 3.0.0
-setuptools == 15.1
+setuptools >= 15.1
 pyenchant == 1.6.6
-scipy == 0.14.0
-scikit-learn == 0.15.2
+scipy >= 0.14.0
+scikit-learn >= 0.15.2
```

Comment:
Good questions.  I think I can actually lock the versions now that I'm using miniconda to download and install the modules.  The reason I changed some of them to >= was to make it possible to install the available precompiled py modules, which are sometimes a few versions away.

Lemme play with the versions and see whether I can find a better compromise.